package-lock.json

```diff
@@ -1833,7 +1833,22 @@
             "resolved": "https://registry.npmjs.org/lambdafs/-/lambdafs-2.0.3.tgz",
             "integrity": "sha512-5YWwZA/QKk09GdfcJ/ABVO+bpFoGlnTBa5jmyM8Kt9yIzl2lDDVBPLK+Aenq2UEcuDpqxXIYI5zLB7VZNepTrg==",
             "bundleDependencies": [
-                "tar-fs"
+                "tar-fs",
```

Comment:
I don't know what all these dependencies are, but I am seeing them on my diffs locally too, so I think it's good to merge them.

Reply:
I just did a `npm install` and that popped up.  I'm sure we are all running slightly different versions of `npm`